Dvg/post 120 update (#14)
lamps-wg/cms-kyber: draft-ietf-lamps-cms-kyber.md

> All identifiers used by ML-KEM in CMS are defined elsewhere but reproduced here for convenience:

Reviewer: ```suggestion
All identifiers used to indicate ML-KEM within CMS are defined elsewhere but reproduced here for convenience:
```
